components/l10n/browser/locale_helper.h

```diff
@@ -22,8 +22,7 @@ class LocaleHelper {
 
   static LocaleHelper* GetInstance();
 
-  void set_for_testing(
-      LocaleHelper* locale_helper);
+  static void SetLocaleHelperForTesting(LocaleHelper* locale_helper);
```

Comment:
nit: maybe remove LocaleHelper as would be less verbose with `LocaleHelper::SetForTesting`

Reply:
done
https://github.com/brave/brave-core/pull/13552/commits/b920d380c5e65e5bcbf7c67b33dbee34d93a851c